config.json

```diff
@@ -172,6 +172,14 @@
           "math"
         ]
       },
+      {
+        "slug": "nucleotide-count",
+        "name": "Nucleotide Count",
+        "uuid": "06ea660b-3869-46df-8fd3-a0651b66ed54",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2
```

Comment:
This is the first exercise where the user has to throw an error. They'll probably need to research that a bit. I'd suggest bumping the difficulty a notch, but leave it in this order.

Reply:
I can do an instructions append like Python does to briefly introduce the concept of errors and point students to resources like the Dart docs

Reply:
That would be fantastic.

Reply:
Still working on the instructions.append file, but for now, I'm switching to using exceptions. I figure that should be a little simpler for the reader to grasp conceptually. In my reading, an exception boils down to "something unexpected happened, but it's not the programmer's fault". A string containing something besides `A`, `C`, `G`, and `T` fits that because the caller provided something invalid when our code only handles those four letters by design. An error is more like there's an issue in the code that needs to be addressed like we're dividing a number the caller provided by zero.

By having the student create a custom exception that we check for, we can start to show that a more specific type makes handling exceptions easier.